scripts/verify_wtt_deployment/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "verifytokenbridgedeployment",
```

Comment:
it's better to use the `@wormhole-foundation` namespace here, even if the package never gets published, to avoid name squatters messing with package resolution

Reply:
good point, updated